bootstrap/node.go

```diff
@@ -45,6 +46,7 @@ func NewNode(
 		constantConsensusLeader,
 		activeConsensusAlgo,
 		benchmarkConsensusRoundRetryIntervalMillisec,
+		blockSyncCommitTimeoutMS,
```

Comment:
Can we have the name consistent with the line above? Choose one format and change both to match :)